app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionDetailFragment.java

```diff
@@ -64,14 +65,35 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         AppUtil.setTaskDescription(activity, session.title, ContextCompat.getColor(activity, session.category.getVividColorResId()));
     }
 
+    // Receive all viewModel's properties changed.
+    private final Observable.OnPropertyChangedCallback onPropertyChanged = new Observable.OnPropertyChangedCallback() {
```

Comment:
Let's consider about ViewModel lifecycle.
It should be same as view lifecycle.

What does "view" mean in Android?
It is Activity or Fragment.
You should determine policy A."activiy-viewmodel" or B."fragment-viewmodel".

I usually choice "A". because I dislike Fragment. :)

In choice of "B", using OnPropertyChanged in Fragment is no problem. Because these are same life cycle.

If you can't close "same lifecycle world", then you can use EventBus.(ex. Activity receive signals from fragment-depend ViewModel.)